src/widget/analog-scale/analog-scalepicker.js

```diff
@@ -133,12 +133,14 @@ class AnalogScaleWidget extends RangeWidget {
     get props() {
         const props = this._props;
         props.touch = support.touch;
+        props.showScale = props.appearances.includes( 'show-scale' );
         props.min = isNumber( this.element.getAttribute( 'min' ) ) ? this.element.getAttribute( 'min' ) : 0;
         props.max = isNumber( this.element.getAttribute( 'max' ) ) ? this.element.getAttribute( 'max' ) : 100;
-        props.step = isNumber( this.element.getAttribute( 'step' ) ) ? this.element.getAttribute( 'step' ) : 1; //( props.type === 'decimal' ? 0.1 : 1 );
+        props.step = isNumber( this.element.getAttribute( 'step' ) ) ? this.element.getAttribute( 'step' ) : ( props.showScale ? 10 : 1 ); //( props.type === 'decimal' ? 0.1 : 1 );
```

Comment:
This is your proxy "range" detection, right? Hmm, that seems quite bullet proof.

However, what if a user uses `range` without `step` (not mandatory) and with `show-scale`? (I haven't tested this)

Reply:
Tested it and the widget will just show 'usual' range widget with no step-scale visible.